coreferee/tendencies.py

```diff
@@ -23,13 +23,6 @@ def __init__(
     ):
         self.rules_analyzer = rules_analyzer
         self.vectors_nlp = vectors_nlp
-        if self.vectors_nlp.vocab[rules_analyzer.random_word].has_vector:
```

Comment:
This turns out to be obsolete code from the time when Coreferee used Keras rather than Thinc. It can be removed and does not need replacing elsewhere in the codebase.